cypress/e2e/item/share/shareItem.cy.ts

```diff
@@ -126,4 +128,15 @@ describe('Share Item', () => {
       expect(url).to.include(item.id);
     });
   });
+
+  it('Share Item with QR Code', () => {
+    cy.setUpApi({ ...SAMPLE_PUBLIC_ITEMS });
+    const item = SAMPLE_PUBLIC_ITEMS.items[0] as any;
```

Comment:
You don't need this cast here. In general it is better to not cast to any when not necessary.

```suggestion
    const item = SAMPLE_PUBLIC_ITEMS.items[0];
```